e2e/support/test_support.go

```diff
@@ -445,6 +445,17 @@ func IntegrationLogs(ns, name string) func() string {
 	}
 }
 
+func TailedLogs(ns, name string) func() string {
```

Comment:
We should parametrize the number of lines to tail IMO. It is a quick win very useful for future usage. Additionally some quick comment as a documentation would be nice as well.